doc/source/widgets/multiaxisplotwidget.rst

```diff
@@ -0,0 +1,7 @@
+MultiAxisPlotWidget
+===============
```

Comment:
needs more `=` characters to line up w/ the line above

Reply:
done